Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -478,16 +519,18 @@ - (ADBMediaHeartbeatConfig *)createConfig:(SEGTrackPayload *)payload
     NSDictionary *properties = payload.properties;
     NSDictionary *options = payload.integrations[@"Adobe Analytics"];
 
-    bool sslEnabled = false;
+    BOOL sslEnabled = false;
     if (self.settings[@"ssl"]) {
         sslEnabled = true;
     }
 
-    bool debugEnabled = false;
+    BOOL debugEnabled = false;
     if (options[@"debug"]) {
         debugEnabled = true;
     }
 
+    //    self.config = [[ADBMediaHeartbeatConfig alloc] init];
```

Comment:
Does leaving this commented like this serve a purpose?

Reply:
oh just missed this